### ISSUE TITLE
sesv2 send_email fails if property 'Text' is not used in Content/Simple/Body

### ISSUE THREAD

**Issue by martinezedgar** [getmoto/moto#6929]
Using moto 4.2.6, boto3 1.28.29 and python 3.11.4

If you use the send_email method without the 'Text' property in Content/Simple/Body  like this:

```
sesv2.send_email(
            FromEmailAddress=source,
            Destination={
                'ToAddresses': [destination]
            },
            Content={
                'Simple': {
                    'Subject': {
                        'Data': subject,
                        'Charset': 'UTF-8'
                    },
                    'Body': {
                        'Html': {
                            'Data': message,
                            'Charset': 'UTF-8'
                        }
                    }
                },
            },
        )
```

Instead of this:

```
sesv2.send_email(
            FromEmailAddress=source,
            Destination={
                'ToAddresses': [destination]
            },
            Content={
                'Simple': {
                    'Subject': {
                        'Data': subject,
                        'Charset': 'UTF-8'
                    },
                    'Body': {
                        'Html': {
                            'Data': message,
                            'Charset': 'UTF-8'
                        },
                        'Text': {
                            'Data': message,
                            'Charset': 'UTF-8'
                        }
                    }
                },
            },
        )
```

The following error occurs:

```
self = <moto.sesv2.responses.SESV2Response object at 0x11f1d8310>

    def send_email(self) -> str:
        """Piggy back on functionality from v1 mostly"""
    
        params = json.loads(self.body)
        from_email_address = params.get("FromEmailAddress")
        destination = params.get("Destination", {})
        content = params.get("Content")
        if "Raw" in content:
            all_destinations: List[str] = []
            if "ToAddresses" in destination:
                all_destinations = all_destinations + destination["ToAddresses"]
            if "CcAddresses" in destination:
                all_destinations = all_destinations + destination["CcAddresses"]
            if "BccAddresses" in destination:
                all_destinations = all_destinations + destination["BccAddresses"]
            message = self.sesv2_backend.send_raw_email(
                source=from_email_address,
                destinations=all_destinations,
                raw_data=base64.b64decode(content["Raw"]["Data"]).decode("utf-8"),
            )
        elif "Simple" in content:
            message = self.sesv2_backend.send_email(  # type: ignore
                source=from_email_address,
                destinations=destination,
                subject=content["Simple"]["Subject"]["Data"],
>               body=content["Simple"]["Body"]["Text"]["Data"],
E               KeyError: 'Text'

.venv3114/lib/python3.11/site-packages/moto/sesv2/responses.py:49: KeyError
```

Up to moto version 4.2.5 there was no problem in not using the 'Text' property, just like in boto3.

**Comment by martinezedgar** [comment]
Hi @bblommers,

Yes, it is already solved in 4.2.7.dev11, thank you very much for the help.

**Comment by bblommers** [comment]
Happy to hear that!
